source/features/quick-repo-deletion.tsx

```diff
@@ -103,6 +105,7 @@ async function start(buttonContainer: HTMLDetailsElement): Promise<void> {
 			{action: false},
 		);
 		select('.application-main')!.remove();
+		await cache.delete(getCacheKey());
```

Comment:
Could this throw an error if the key doesn't exist in the cache? The user might have disabled `forked-to` for example.

Reply:
I've tested this beforehand and it does nothing if there's no such key.